contracts/Token/OpThales_L1.sol

```diff
@@ -1,16 +1,42 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.6.10;
 
-import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-// import { IL2StandardERC20 } from "@eth-optimism/contracts/libraries/standards/IL2StandardERC20.sol";
-import { iOVM_L1StandardBridge } from "@eth-optimism/contracts/iOVM/bridge/tokens/iOVM_L1StandardBridge.sol";
-
-contract OpThales is ERC20 {
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {iOVM_L1StandardBridge} from "@eth-optimism/contracts/iOVM/bridge/tokens/iOVM_L1StandardBridge.sol";
 
+contract OpThales is ERC20, Ownable {
+    string public __name = "Optimistic Thales Token";
+    string public __symbol = "opTHALES";
+    uint8 public constant __decimals = 18;
     uint private INITIAL_TOTAL_SUPPLY = 100000000;
 
-    constructor () public ERC20("Optimistic Thales Token", "opTHALES") {
+    event NameChanged(string name);
+    event SymbolChanged(string symbol);
+
+    function name() public view override returns (string memory) {
+        return __name;
+    }
+
+    function symbol() public view override returns (string memory) {
```

Comment:
if this is the getter then __symbol should be private

Reply:
OK